interfaces/IPrivacy.h

```diff
@@ -0,0 +1,245 @@
+#pragma once
+
+#include "Module.h"
+
+namespace WPEFramework
+{
+    namespace Exchange
+    {
+        // @json 1.0.0 @text:keep
+        struct EXTERNAL IPrivacy : virtual public Core::IUnknown
+        {
+            enum
+            {
+                ID = ID_PRIVACY
+            };
+
+            virtual ~IPrivacy() override = default;
+
+            // @event
+            struct EXTERNAL INotification : virtual public Core::IUnknown
+            {
+                enum
+                {
+                    ID = ID_PRIVACY_NOTIFICATION
+                };
+
+                // @text onAllowACRCollectionChanged
```

Comment:
I think, @sebaszm to confirm, but there is no need to use the @text to change the PascalCasing to camelCasing on JSONRPC. The Thunder framework (at least on R5.2) and further will except both for incoming and I think (again @sebaszm to confirm) send out the camelCase variant for events...
So if the intention is for this interface to be exposed on R5.2 or higher, the @text is no needed..

Reply:
That's correct, with small remark: acronyms will also be converted according to camelCase convention. So if uppercase acronyms (like "ACR" here) need to be kept uppercase then (a)text is still required in such cases.